src/code/z_sub_s.c

```diff
@@ -36,9 +36,120 @@ EnDoor* SubS_FindDoor(GlobalContext* globalCtx, s32 unk_1A5) {
     return door;
 }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013A860.s")
+Gfx* SubS_DrawTransformFlexLimb(GlobalContext* globalCtx, s32 limbIndex, void** skeleton, Vec3s* jointTable,
+                                OverrideLimbDraw overrideLimbDraw, PostLimbDraw postLimbDraw,
+                                TransformLimbDraw transformLimbDraw, Actor* actor, Mtx** mtx, Gfx* gfx) {
+    StandardLimb* limb;
+    Gfx* newDList;
+    Gfx* limbDList;
+    Vec3f pos;
+    Vec3s rot;
+
+    Matrix_StatePush();
+    limb = (StandardLimb*)Lib_SegmentedToVirtual(skeleton[limbIndex]);
```

Comment:
Since `Lib_SegmentedToVirtual` returns a `void*` the cast shouldn't be necessary.
```suggestion
    limb = Lib_SegmentedToVirtual(skeleton[limbIndex]);
```